Fhi.HelseId.Web/AuthHeaderHandler.cs

```diff
@@ -54,16 +55,21 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
         }
 
         var ctx = _contextAccessor.HttpContext ?? throw new NoContextException();
+        var endpoint = ctx.GetEndpoint();
+        var allowAnonymous = endpoint?.Metadata.GetMetadata<AllowAnonymousAttribute>();
+
         _logger.LogTrace("{class}.{method} - Starting", nameof(AuthHeaderHandler), nameof(SendAsync));
         var token = await ctx.GetTokenAsync(OpenIdConnectParameterNames.AccessToken);
 
-        if (token == null)
+        if (token == null && allowAnonymous == null)
         {
-            _logger.LogError("{class}.{method} No access token found in context. Make sure you have added the" +
+            _logger.LogError(
+                "{class}.{method} No access token found in context. Make sure you have added the" +
                 " AddTokenManagement() to your Startup.cs",
```

Comment:
We can remove {class}.{method} from the log statements as this is already available in other forms, not important for this PR